src/vs/editor/contrib/copyPaste/browser/copyPasteController.ts

```diff
@@ -69,9 +70,12 @@ export class CopyPasteController extends Disposable implements IEditorContributi
 	}
 
 	private arePasteActionsEnabled(model: ITextModel): boolean {
-		return this._configurationService.getValue('editor.experimental.pasteActions.enabled', {
-			resource: model.uri
-		});
+		if (this._configurationService.getValue('editor.experimental.pasteActions.enabled', { resource: model.uri })) {
```

Comment:
Should this setting name change?

Reply:
This is the general setting for pasting. This one is still experimental and will not be finalized this iteration

We only want to enable this feature for our builtin ipynb extension, which has it's own setting to control pasting just for ipynb files